RELEASING.md

```diff
@@ -56,3 +53,12 @@ If review is not required, commit directly to `dev`.
    ```
 
 5. Announce new release in [Slack](https://kotlinlang.slack.com).
+
+# API reference documentation
+
+The [API reference documentation](https://kotlinlang.org/api/kotlinx.serialization/) is built and deployed automatically
+for every commit in `master`, typically within the same day.
```

Comment:
Please add link to TC so team members can easily find it.

Also, you haven't explained the relation between `master` builds and tag. I think you need to include sentence like 'Despite automatic builds, it is still important to update `KOTLINX_SERIALIZATION_RELEASE_TAG` because it is used to display a version number on the website.'

Reply:
At the time of writing it, I didn't actually know why it was needed :sweat_smile:  Tried to capture what I found out :+1: